opencensus/exporters/stats/stackdriver/stackdriver_exporter.h

```diff
@@ -36,6 +36,12 @@ struct StackdriverOptions {
 
   // The RPC deadline to use when exporting to Stackdriver.
   absl::Duration rpc_deadline = absl::Seconds(5);
+
+  // If empty, the exporter will use "custom.googleapis.com/opencensus"
+  std::string metric_domain;
+
+  // If empty, the exporter will use "global"
+  std::string resource_type;
```

Comment:
Do we have a logic to determine how the resource labels are set?

Reply:
Thoughts on using [MonitoredResource proto](https://github.com/googleapis/googleapis/blob/07244bb797ddd6e0c1c15b02b4467a9a5729299f/google/monitoring/v3/metric.proto#L60) directly in this config?  The [definition](https://github.com/googleapis/googleapis/blob/d84bde8698967faa93ab56cc7132965f6be534ad/google/api/monitored_resource.proto#L86) already has comments around its usage as well.

Otherwise we also need zone and instance_id here.

Reply:
I've split this change out of this PR. I'll do a followup PR with the proto as suggested.